api/config/config.py

```diff
@@ -6,8 +6,8 @@
 
 # Sample config for PostgreSQL Database
 POSTGRES = {
-    'user': 'badgeyay',
-    'pw': 'badge',
+    'user': 'postgres',
```

Comment:
this is not good practice, @djmgit this [PR](https://github.com/fossasia/badgeyay/pull/741) will solve the issue

Reply:
well, it is just a config. It does not share any public key or keys so I suppose it is quite okay :/

Reply:
As mentioned it is just for the LOCAL environment and posthresql creates posthresql user as default and there is no harm in it as in the production we are using heroku postgres and cloud psql. So I think it is not a big issue